formulas/container/install.sls

```diff
@@ -36,10 +36,21 @@ container_packages:
       - kata-proxy
       - kata-shim
 
-pymysql_sa:
-  pip.installed:
-    - bin_env: '/usr/bin/pip3'
-    - reload_modules: true
```

Comment:
This still needs installed, but since it is installed no matter what, it should be removed from the conditional statement and place outside of it